podkrepi-bg/frontend
# Add stripe donations

## Motivation and context

Add stripe donations

Closes #375 

## Screenshots:

Before|After
---|---
![image](https://user-images.githubusercontent.com/893608/138724162-f9943746-c035-438b-81d2-2f044a08de46.png)|![image](https://user-images.githubusercontent.com/893608/139065473-aabe51a8-cd25-4f00-9a79-b686f83c9044.png)


## Testing

1. Go to campaign page
2. See donation prices on the right
3. Click on donation price
4. See stripe checkout page
5. Fill credit card data with `4242 4242 4242 4242` `12/24`  and click Pay
6. Be back on the website after successful payment


**New dev dependencies**:

- `stripe` : usage of Stripe types
